README.md

```diff
@@ -0,0 +1,56 @@
+# browser-logging
+
+Listens for logs from [browser-logging-client](https://www.npmjs.com/package/browser-logging-client).
```

Comment:
Not sure if this is the right way around? Both client and server projects say 'Listens for logs from {the other one}' so one of them is wrong!